osi_object.proto

```diff
@@ -865,6 +865,11 @@ message MovingObject
             // Vehicle is a wheelchair.
             //
             TYPE_WHEELCHAIR = 15;
+			
+            // Vehicle is a stand-up or kickboard scooter, including motorized versions.
+			// This vehicle type accounts for the following test dummies: https://www.messring.de/de/produkte/aktive-sicherheit/astera-e-scooter/ and https://www.4activesystems.at/4activeps-ekickboard-scooter
```

Comment:
When you're using references you should comply with the [contribution guidelines concerning references](https://opensimulationinterface.github.io/osi-documentation/#top-guidelines_for_creating_literary_references). The referenced websites may not be reliably available in the future. Maybe the description in line 868 alone is even enough to know what is meant?

Also, the build tests fail again because there are some additional tabs before the "//"-comment in line 870.